### ISSUE TITLE
Gameplay feature: Lightning Round

### ISSUE THREAD

**Issue by GeorgeCiesinski** [GeorgeCiesinski/poke-guesser-bot#34]
(Issue should be expanded before starting changes)

# Describe the new feature
Add a new command to start a lightning round, where a new pokemon will be generated upon each successful guess. This game mode would be toggled on/off, or would automatically expire after x number of loops, where x is provided alongside the command. 

## Planning
1. Add a command for the lightning round with an optional loops parameter: 

### Example: Toggle Lightning round on/off
`!lightning`

### Example: Turn on Lightning round for 5 loops
`!lightning 5`

### Current Mode
During lightning round, a part of the message should show a helpful icon like lightning strike, and inform the players that the current mode is lightning round in order to prevent it from being left on longer than intended.

## Organization

Game modes might need to be organized into a new file that imports necessary functions as more game modes might be added eventually.

**Comment by Wissididom** [comment]
Maybe we could use :cloud_lightning: as symbol for the lightning round bc that's the only emote i found in discord with a lightning in it. I'm gonna think about what to put in the extra file. Maybe export an gamemodes.exploreLightning method doing the processing and maybe add an optional parameter to util.embedReply for the lightning symbol. Just an idea i have so far about organizing of the gamemode.

**Comment by Wissididom** [comment]
I just had the thought to add something like !cancellightning or so. Example:
A lightning round for let's say 10 rounds has been started and the Bot-Admin wants to end after 5 rounds or so.

**Comment by GeorgeCiesinski** [comment]
Should we close this issue?

**Comment by Wissididom** [comment]
I would say yes. Also, because it would have been automatically closed if we'd use the master branch as the base branch for PRs.
Improvements for the feature can always be new issues.
I think the improvements that can be looked into are the possibility to cancel and an icon or emoji that shows that it is a lightning round. Creating a GameModes file I would be doing as a separate thing once we get other game mode ideas, if we get some.

**Comment by GeorgeCiesinski** [comment]
I think it should still close issues when merged into develop branch if you include `fixes #34` in the commit message. I am not sure about PR though. That works when I merge branches normally.

**Comment by Wissididom** [comment]
I never put those command words etc. into the commit messages. I always used the PR description/comments, where it closed them everywhere I merged those PRs except here, and I think I've read somewhere that it only closes if the commits are merged into the default branch and if that is the case I think the easiest way to solve that would be to simply make the develop branch the default branch of this repository.

Found something: https://github.blog/2013-01-22-closing-issues-via-commit-messages/
